.github/workflows/_run_e2e_test.yml

```diff
@@ -0,0 +1,43 @@
+name: Common Steps
+
+on:
+  workflow_call:
+    inputs:
+      url:
+        required: true
+        type: string
+      working-directory:
+        required: false
+        type: string
```

Comment:
No need, as E2E tests run exclusively in `frontend/packages/e2e`.

Reply:
Yes.🙏 I've removed working-directory: section.
https://github.com/liam-hq/liam/pull/660/commits/a9e47e3099e4ff25168e070cfc04b7d6162c7d12